http/sys_health.go

```diff
@@ -131,4 +146,6 @@ type HealthResponse struct {
 	Standby       bool   `json:"standby"`
 	ServerTimeUTC int64  `json:"server_time_utc"`
 	Version       string `json:"version"`
+	ClusterName   string `json:"cluster_name"`
```

Comment:
Let's `omitempty` for these two values unless we hear a complaint about why we shouldn't. We can always add, but it's harder to subtract later.